src/game/Maps/ScriptCommands.cpp

```diff
@@ -1195,5 +1195,28 @@ bool Map::ScriptCommand_RemoveItem(ScriptAction& step, Object* source, Object* t
 
     pPlayer->DestroyItemCount(step.script->createItem.itemEntry, step.script->createItem.amount, true);
 
+    return false;
+}
+
+// SCRIPT_COMMAND_REMOVE_OBJECT (41)
+bool Map::ScriptCommand_RemoveGameObject(ScriptAction& step, Object* source, Object* target)
+{
+    Unit* pUser = nullptr;
+    GameObject *pGo = nullptr;
+
+    if (!((pUser = ToUnit(source)) || (pUser = ToUnit(target))))
```

Comment:
What is the point of pUser when you are not using it. Looks like a copy-paste mistake?

Reply:
Yeah fixed it.